### DISCUSSION TITLE
Issue with PagerOptions packaging

### DISCUSSION THREAD

**Discussion by pbull001** [OrchardCMS/OrchardCore#12628] [General]
### To Reproduce
1. Reference OrchardCore.Navigation nuget package 1.4.0
2. Create a new controller
3. Define a class variable of the type Pager (works fine)
4. Define a class variable of type PagerOptions (see missing type or namespace reference intellisense error)

### Expected behavior
Given both Pager and PagerOptions are defined in the same project and namespace I would expect the PagerOptions class type to pull through the same way

### Screenshots

<img width="627" alt="Capture" src="https://user-images.githubusercontent.com/18185015/194993246-abc70c8e-5a21-40e7-8e27-c604904b0cb4.PNG">

**Comment by MikeAlhayek** [comment]
@pbull001 you seem to be using version 1.4.0. PagerOptions was introduced in 1.5.0-preview. Either use the 1.5.0-preview or wait for the final release of 1.5 then upgrade your project.
